projects/effective-proposal-framework-and-delius/README.md

```diff
@@ -1,3 +1,13 @@
-# effective-proposal-framework-and-delius
+# Effective Proposal Framework and Delius
 
 A probation integration service to share data between Ctrl-O's 'Effective Proposal Framework' system and the Delius probation case management system.
+
+## Context Map
+
+![Context Map](./tech-docs/source/img/epf-and-delius-context-map.svg)
+
+## Interfaces
+
+### HMPPS External API
+
+The HMPPS External API provides the gateway, access control and client relationship management for this API.
```

Comment:
Is this meant for the `external-api-and-delius` integration docs? Or does EPF use the HMPPS External API too?

Reply:
Ctrl-O are going to access this via the HMPPS External API to give us a consistent gateway for external clients